grails-plugin-gsp/src/main/groovy/org/grails/plugins/web/taglib/FormTagLib.groovy

```diff
@@ -548,6 +549,57 @@ class FormTagLib implements ApplicationContextAware, InitializingBean, TagLibrar
         out << '/>'
     }
 
+    /**
+     * Creates a submit button that submits to an action generated by the various link attributes.<br/>
+     *
+     * &lt;g:formActionSubmit action="myaction" value="Submit"/&gt;<br/>
+     * &lt;g:formActionSubmit controller="myctrl" action="myaction" value="ButtonName"/&gt;<br/>
+     *
+     * @attr action the name of the action to use in the submit action, if not specified the default action will be linked
+     * @attr controller the name of the controller to use in the submit action, if not specified the current controller will be linked
+     * @attr id the id attribute of the formActionSubmit tag
```

Comment:
I copied the same attributes that were on the form, but technically the full link attributes are supported, other than id & method.  Note: I wanted to maintain the same syntax as the `g:form` so that's why I didn't use elementId.  I'll go ahead and copy several of the other attr documentation over for clarity & to match the user guide doc.